boilerplates/react-union-boilerplate-ssr-basic/README.md

```diff
@@ -0,0 +1,98 @@
+# React Union - Basic boilerplate
```

Comment:
Oh yeah, I forgot to fix this file 😄I'll do it alongside the tests and documentation fixes.